### DISCUSSION TITLE
Creating new profile asks for vault password. Not entering results in no error.

### DISCUSSION THREAD

**Discussion by timboormans** [Eugeny/tabby#9014] [Q&A]
Hi,

Not sure if this is correct behavior, but on Windows when using Tabby > Connections > New profile > SSH connection sometimes asks for the Vault Password to be entered. Not all the times though. If i then press the Esc(ape) button on my keyboard then it closes the password popup and just continues as if the correct password was given. I am not sure why the popup occurred, and so I am not sure if this is the correct resulting behavior.

Edit: I also experience this same application behavior when going through the Profiles > Advanced > Any of the setting popup hyperlinks. So for example "SSH" then i get a popup "Vault password", then i press Esc and i can freely continue, make changes, and it of course saves my changes. But not sure why that Vault Password popup is popping up on every click.

**Comment by TooMuhtsh** [comment]
Those popups appear whenever a settings screen needs to read a vault secret. If they get in your way, tabby-better-vault answers them automatically from the OS keychain — you set it up once and the prompts are gone. Opt-in, native prompt as fallback.
